CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.35.0] - 2021-11-5
```

Comment:
this should be a patch, so 2.34.1 instead

Reply:
we were thinking of a 2.34.1 "ghost version" with all the functionality, but no documentation ( #868 ) to be released now, and a 2.35.0 (this) version with the documentation to make it official to the users as part of the December release.

What do you think would be the best aproach?

Reply:
Hmm, I think it would be better to add this documentation as part of the functionality so that the feature change is complete in itself. So maybe close this PR and merge it to the other PR and version that one to 2.35.0?

Reply:
This was because we need to make use of the official SDK before release time without announcing it. So that is the reason why we have 2 PRs. Is this a wrong approach?

Reply:
Don't need to bump version at all to update docs.